spec/features/admin/index_spec.rb

```diff
@@ -6,12 +6,28 @@
   end
 
   describe "Admin Dashboard Display" do
-    
     # User Story 19
     it "displays a header indicating that it is the admin dashboard" do
       visit admin_path
 
       expect(page).to have_content("Admin Dashboard")
     end
+
+    # User Story 20
+    it "displays a link to the admin merchants index page" do
+      visit admin_path
+
+      expect(page).to have_link("Merchants", href: admin_merchants_path)
```

Comment:
I didn't realize that you can add the path in the "have_link" part. The href:...
Pretty cool!